randovania/games/dread/json_data/Dairon.txt

```diff
@@ -2797,28 +2785,34 @@ Extra - asset_id: collision_camera_033
 
 > Room Center; Heals? False
   * Layers: default
+  > Dock to Shinespark Tutorial
+      Lay Bomb or Lay Power Bomb
   > Door to Ammo Recharge Station
       Trivial
   > Pickup (Missile+ Tank)
       All of the following:
           Speed Booster
           Any of the following:
               Lay Bomb or Lay Power Bomb
-              After Dairon - Storm Gate Speed Blocks Destroyed and Walljump (Beginner)
-  > Tile Group (BOMB)
-      Morph Ball
+              After Dairon - Storm Gate Speed Blocks Destroyed and Movement (Intermediate)
```

Comment:
I'm guessing there's space to charge a speedbooster from the water room in the water? Either way, I would add a comment on where the speedboost is charged for clarification

Reply:
This is for charging the speed where the Storm Gate was, but its very tight.